default-application-catalog/cilium-app.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: apps.kubermatic.k8c.io/v1
+kind: ApplicationDefinition
+metadata:
+  name: cilium
```

Comment:
This shouldn't be shipped, it's already available as a CNI "System Application".

Reply:
As discussed, removed from the list.